internal/worker/rundynamic.go

```diff
@@ -137,6 +161,15 @@ func RunDynamicAnalysis(ctx context.Context, pkg *pkgmanager.Pkg, sbOpts []sandb
 		analysisCmd = dynamicanalysis.DefaultCommand(pkg.Ecosystem())
 	}
 
+	// Adding environment variable baits
```

Comment:
Could you expand the comments a little bit (one or two lines) to describe what the access keys mean? Or maybe just include the link to [the docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)